code/Controllers/CMSPageHistoryController.php

```diff
@@ -247,9 +247,9 @@ public function getEditForm($id = null, $fields = null, $versionID = null, $comp
         /** @var Tab $mainTab */
         $mainTab = $fields->fieldByName('Root.Main');
         $mainTab->unshift(
-            new LiteralField('CurrentlyViewingMessage', ArrayData::create(array(
+            LiteralField::create('CurrentlyViewingMessage', ArrayData::create(array(
                 'Content' => DBField::create_field('HTMLFragment', $message),
-                'Classes' => 'notice'
+                'Classes' => 'alert alert-info'
```

Comment:
Should we just move these to the template? I don’t see any advantage to keeping the classes in PHP

Reply:
The only drawback I see is that it can’t be reused for other alert types then, however this is the only place it’s used currently

Reply:
Truedat, let’s leave it then